src/containers/explore/EntityDataContainer.js

```diff
@@ -0,0 +1,150 @@
+/*
+ * @flow
+ */
+
+import React, { useEffect } from 'react';
+
+import styled from 'styled-components';
+import { List, Map } from 'immutable';
+import {
+  AppContentWrapper,
+  Spinner,
+  Typography
+} from 'lattice-ui-kit';
+import {
+  DataUtils,
+  ReduxUtils,
+  ValidationUtils,
+  useRequestState
+} from 'lattice-utils';
+import { useDispatch, useSelector } from 'react-redux';
+import type { FQN, Organization, UUID } from 'lattice';
+import type { RequestState } from 'redux-reqseq';
+
+import EntityNeighborsContainer from './EntityNeighborsContainer';
+import {
+  EXPLORE_ENTITY_DATA,
+  exploreEntityData,
+  exploreEntityNeighbors
+} from './actions';
+
+import {
+  CrumbItem,
+  CrumbLink,
+  Crumbs,
+  LinkButton
+} from '../../components';
+
+import { FQNS } from '../../core/edm/constants';
+import {
+  ENTITY_NEIGHBORS_MAP,
+  EXPLORE,
+  SELECTED_ENTITY_DATA
+} from '../../core/redux/constants';
+import { selectOrgDataSet, selectOrganization } from '../../core/redux/selectors';
+import { Routes } from '../../core/router';
+import { clipboardWriteText } from '../../utils';
+import { EntityDataGrid } from './components';
+
+const { getPropertyValue } = DataUtils;
+const { isPending } = ReduxUtils;
+const { isValidUUID } = ValidationUtils;
+
+const FlexContainer = styled.div`
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 30px;
+`;
+
+type Props = {
+  dataSetDataRoute ?:string;
+  dataSetId :UUID;
+  entityKeyId :UUID;
+  isModal ?:boolean;
+  organizationId :UUID;
+  organizationRoute ?:string;
+};
+
+const EntityDataContainer = ({
+  dataSetDataRoute,
+  dataSetId,
+  entityKeyId,
+  isModal = false,
+  organizationId,
+  organizationRoute,
+} :Props) => {
+  const dispatch = useDispatch();
+
+  const linkString = `https://openlattice.com/orgs/#${
+    Routes.ENTITY_DETAILS
+      .replace(Routes.DATA_SET_ID_PARAM, dataSetId)
+      .replace(Routes.ENTITY_KEY_ID_PARAM, entityKeyId)
+      .replace(Routes.ORG_ID_PARAM, organizationId)
+  }`;
```

Comment:
Please don't hardcode `https://openlattice.com/orgs`.

Reply:
What's the best way to handle this?

Reply:
Use `window.location`:

https://developer.mozilla.org/en-US/docs/Web/API/Window/location
https://developer.mozilla.org/en-US/docs/Web/API/Location

<img width="591" alt="Screen Shot 2021-06-10 at 12 02 01 PM" src="https://user-images.githubusercontent.com/440299/121582318-b5376e00-c9e3-11eb-8baa-2e4a94d48ff3.png">